types.go

```diff
@@ -67,14 +67,15 @@ type Annotations string
 
 // Supported annotations param values
 const (
-	AnnotationsTrue        Annotations = "true"
-	AnnotationsFalse       Annotations = "false"
-	AnnotationsNodes       Annotations = "nodes"
-	AnnotationsDistance    Annotations = "distance"
-	AnnotationsDuration    Annotations = "duration"
-	AnnotationsDatasources Annotations = "datasources"
-	AnnotationsWeight      Annotations = "weight"
-	AnnotationsSpeed       Annotations = "speed"
+	AnnotationsTrue             Annotations = "true"
+	AnnotationsFalse            Annotations = "false"
+	AnnotationsNodes            Annotations = "nodes"
+	AnnotationsDistance         Annotations = "distance"
+	AnnotationsDuration         Annotations = "duration"
+	AnnotationsDatasources      Annotations = "datasources"
+	AnnotationsWeight           Annotations = "weight"
+	AnnotationsSpeed            Annotations = "speed"
+	AnnotationsDurationDistance Annotations = "duration,distance"
```

Comment:
Why do you need that?
Maybe add a function to combine multiple annotations in a request?

Reply:
I can do the function. There's a request in TabeService which combines duration and distance, and returns it in the response. That's the reason I need it.
I thought this would be more simpler, since this is considered like a 3rd option for that param in request.

Reply:
I think you could add a function:
```
func (a Annotations) Concat(b Annotations) Annotations {
    return a + "," + b
}
```
and use it in your code as well:
```
AnnotationsDuration.Concat(AnnotationsDistance)
```

Reply:
I was thinking, if I add a function which enables to combine multiple annotations, then users of the library can product invalid annotations which cannot be combined.
By defining it like this user is always going to produce valid annotations.
This 'duration,distance' is the only combination in the project osrm docs.
I would create a function, if there were more combinations possible, but since it's only one, I would rather leave it defined as const.